backend/dal/Security/RoleClaim.cs

```diff
@@ -0,0 +1,29 @@
+namespace Pims.Dal.Security
+{
+    public enum RoleClaim
+    {
+        [ClaimName("admin-users")]
+        AdminUsers,
+
+        [ClaimName("admin-users")]
+        AdminRoles,
+
+        [ClaimName("admin-users")]
+        PropertyView,
+
+        [ClaimName("admin-users")]
+        PropertyAdd,
+
+        [ClaimName("admin-users")]
+        PropertyEdit,
+
+        [ClaimName("admin-users")]
+        DisposeRequest,
+
+        [ClaimName("admin-users")]
+        DisposeApprove,
+
+        [ClaimName("admin-users")]
+        SensitiveView
```

Comment:
Quick question: Do every Role Claim is mapped to the same name "admin-users"? Is this correct? Help me understand

Reply:
These enums simply provide a consistent compile-time safe way to get the string value we use in a Keycloak role.  These roles are converted into Identity Claims.

Reply:
This has been fixed and is now an enum called `Permissions`